Minor fix in package exporter

atuljangra opened the pull request:
Summary:
As title.
Saw this while working on another diff.
`storage` won't be defined in the `else` case. But this causes pyre to freak out.

Test Plan: Unit tests.

Differential Revision: D41751229

PaliC reviewed (approved):
lgtm, please lint though

atuljangra committed:
Minor fix in package exporter (#90306)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/90306

As title.
Saw this while working on another diff.
`storage` won't be defined in the `else` case. But this causes pyre to freak out.

Test Plan: Unit tests.

Reviewed By: PaliC

Differential Revision: D41751229

fbshipit-source-id: dda18cd4a61e65a5f58fa588305b26d250ab8580
